test/sdk/nodejs/node_modules/agones

```diff
@@ -0,0 +1 @@
+../../../../sdks/nodejs
```

Comment:
Should this file (and the next one) but added? They appear when I run tests but I'm not sure whether they should be checked in.

Reply:
oops! probably not. Let me go back and remove them.

Should I .gitignore them now, since I'm in there?

Reply:
That sounds like a good idea.